### ISSUE TITLE
Error: Cannot find name 'Map' with version 11.10.0

### ISSUE THREAD

**Issue by jsdevel** [DefinitelyTyped/DefinitelyTyped#33525]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @jessetrinity

If you do not mention the authors the issue will be ignored.

Getting this error now:

```
../../node_modules/@types/node/perf_hooks.d.ts:275:31 - error TS2304: Cannot find name 'Map'.
275         readonly percentiles: Map<number, number>;
                                  ~~~
```

with this config:

```

  
{
  "compilerOptions": {
    "declaration": true,
    "module": "CommonJS",
    "outDir": "dist",
    "sourceMap": true
  },
  "files": [
    "index.ts"
  ]
}
```

my project was compiling just fine with 11.9.6.

**Comment by jessetrinity** [comment]
I believe @SimonSchick is the author.

**Comment by SimonSchick** [comment]
*sigh* this keeps happening... CI never complains when using es6 types but then fails due to missing libs, we probably need to forward declare `interface Map<K, V> {}` same as we do for `Set`.
@jsdevel you should add `lib: ["es2018"]` if you're targeting node 11.
@jessetrinity any suggestions how we can reliably prevent this silliness in the future?

**Comment by jsdevel** [comment]
> we probably need to forward declare interface Map<K, V> {} same as we do for Set.

most likely.  i ran into the same error for `Set` about 6 months ago.

**Comment by jsdevel** [comment]
> @jsdevel you should add lib: ["es2018"] if you're targeting node 11.

in the project i'm working on @SimonSchick , we target every LTS version of node.

**Comment by SimonSchick** [comment]
@jsdevel in that case you should be fine with `es2015`.

**Comment by jessetrinity** [comment]
Consulted with @sandersn about the CI side of this. Will have to run in by @RyanCavanaugh as well. Thanks for bringing this up!

**Comment by jsdevel** [comment]
@sandersn still broken in 11.11.0.  https://travis-ci.org/kogosoftwarellc/open-api/jobs/502762302#L869

**Comment by sandersn** [comment]
@jsdevel The version with the fix is `@types/node@11.10.6`. Is that the version you got for that build? The fix only got merged about 4 hours ago.

cc @SimonSchick, the expert on node types.
